### ISSUE TITLE
Sort by due date

### ISSUE THREAD

**Issue by mattfysh** [moranje/alfred-workflow-todoist#13]
Great workflow, love that I can add a new todo in <1 sec now! Quick question on the `todos` list... is there something I can type or configure to see only overdue + today's tasks? Right now when I type `todos` I'll see a list of (somewhat) arbitrarily-sorted items. Thanks! 😃

**Comment by mattfysh** [comment]
nice-to-have: also sort by priority

**Comment by moranje** [comment]
Hi! Is this still something that would interest you?

**Comment by stphnlwlsh** [comment]
I second this and might be able to take a crack at it.

**Comment by moranje** [comment]
I am working on this myself, but in doing so I refactored a lot of the js code. Now is a bad time to jump into this.

**Comment by moranje** [comment]
I am postponing this feature until next release. Time contrains and all. Thanks for understanding.

Martien

**Comment by iamfreee** [comment]
Any news on this? :)

**Comment by moranje** [comment]
Yeah I have much of the rewritten code finished but I haven't had any time to squash the last bugs and update the test suite between a home make-over, work and my parental duties. I don't have a lot of time to allocate in January. Thanks for helping out.

M

**Comment by moranje** [comment]
Yeah I have much of the rewritten code finished but I haven't had any time to squash the last bugs and update the test suite between a home make-over, work and my parental duties. I don't have a lot of time to allocate in January. Thanks for helping out.

M

**Comment by moranje** [comment]
Yeah I have much of the rewritten code finished but I haven't had any time to squash the last bugs and update the test suite between a home make-over, work and my parental duties. I don't have a lot of time to allocate in January. Thanks for helping out.

M
